src/parse.c

```diff
@@ -24,6 +24,10 @@ t_err	ft_parse_file(char *filename, t_entities *scene, t_cam *cam)
 		ft_free_char_arr(lines);
 		return (ERROR);
 	}
+	if (scene->total != 0)
```

Comment:
What's that do?

Reply:
If we don't have any hittables - which is a correctly formatted .rt file - we want to init the active hittable to -1 to return when we try to change it